recipes-bsp/u-boot/u-boot-beaglev_v2021.04.bb

```diff
@@ -0,0 +1,10 @@
+require recipes-bsp/u-boot/u-boot-common.inc
```

Comment:
I think this file should be called `u-boot-starfive_2021.04.bb`

Reply:
renamed